worlds/pokemon_crystal/data/phone_data.yaml

```
@@ -906,3 +906,27 @@ headbutt_call:
       helps!
     - |
       Peace out.
+
+black_friday_call:
+  caller: out_of_area
+  script:
+    - |
+      Hurry up! Black
+      Friday ends in 2 
+      hours! Don't miss
+    - |
+      out on 30% off on
+      all items, limited
+      to current stock.
+    - |
+      Use code <POKE>30 at
+      checkout in any
+      PokeMart store.
```

```suggestion
      <POKE>MART.
```
Believe it or not, marts are never specifically called this in game, but I think this is how we should refer to them.